sys-boot/gnu-efi/gnu-efi-3.0.6-r1.ebuild

```diff
@@ -61,8 +73,15 @@ efimake() {
 src_compile() {
 	tc-export BUILD_CC AR AS CC LD
 
-	# https://bugs.gentoo.org/607992
-	filter-mfpmath sse
+	if use custom-cflags; then
+		# https://bugs.gentoo.org/607992
+		filter-mfpmath sse
+
+		# https://bugs.gentoo.org/619628
+		append-flags -mno-avx
```

Comment:
I should have known this. Sorry. Just for future reference: Do you have several test installations were you test those things? I was just wondering what the usual procedure is in cases of different architectures were things could essentially be different and/or problematic.